libbeat/processors/add_cloud_metadata/provider_google_gce.go

```diff
@@ -63,11 +77,36 @@ var gceMetadataFetcher = provider{
 						"type": c.Str("machineType"),
 					},
 					"availability_zone": c.Str("zone"),
+					"orchestrator": s.Object{
```

Comment:
These fields should be added to the event only if they are not already set, so it doesn't overwrite the fields set by the metadata providers.

Reply:
👍🏼